From: Marc Leeman <marc.leeman@gmail.com>
Date: Sun, 22 Mar 2026 12:41:52 +0000
Subject: meson missing vmaf_dep variable with vmaf disabled

Last-Update: 2025-12-11
---
 tests/check/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/check/meson.build b/tests/check/meson.build
index 8e13825..5dc9f75 100644
--- a/tests/check/meson.build
+++ b/tests/check/meson.build
@@ -173,7 +173,7 @@ if host_machine.system() != 'windows'
     [['elements/netsim.c'], true ],
     [['elements/shm.c'], not shm_enabled, shm_deps],
     [['elements/unixfd.c'], not gio_unix_dep.found()],
-    [['elements/vmaf.c'], get_option('vmaf').disabled() or not libvmaf_dep.found(), [libvmaf_dep]],
+    [['elements/vmaf.c'], true, ],
     [['elements/voaacenc.c'],
         not voaac_dep.found() or not cdata.has('HAVE_UNISTD_H'), [voaac_dep]],
     [['elements/webrtcbin.c'], not libnice_dep.found(), [gstwebrtc_dep]],
